Library/ENV/scm/git

```diff
@@ -23,6 +23,10 @@ def exec(*args)
   # prevent fork-bombs
   arg0 = args.first
   return if arg0 =~ /^#{F}/i || Pathname.new(arg0).realpath == SELF_REAL
+  if ARGV == %w[--homebrew=print-path]
```

Comment:
Interesting! I'd really like to keep the argument handling as simple as possible and the potential for accidentally matching an argument that was intended for Git itself (however unlikely this is) to a minimum. Particularly, I'd like to avoid supporting `--homebrew=` arguments at arbitrary positions. Would something like the following be sufficient from your point of view?

```ruby
case ARGV.first
when "--homebrew=print-path"
  ARGV.shift
  # React to Homebrew-specific argument.
when "--homebrew=ignore-old-vendor-version"
  ARGV.shift
  # React to Homebrew-specific argument.
end
```